src/extension/config.ts

```diff
@@ -71,6 +71,8 @@ class Config {
 	get cliConsole(): "debugConsole" | "terminal" | "externalTerminal" { return this.getConfig<"debugConsole" | "terminal" | "externalTerminal">("cliConsole", "debugConsole"); }
 	get closeDevTools(): "never" | "ifOpened" | "always" { return this.getConfig<"never" | "ifOpened" | "always">("closeDevTools", "never"); }
 	get closingLabels(): boolean { return this.getConfig<boolean>("closingLabels", true); }
+	get closingLabelsPrefix(): string { return this.getConfig<string>("closingLabelsPrefix", " // "); }
```

Comment:
The default argument passed here should always match the one in the `package.json`. We have code to generate these getters, so they're periodically overwritten with the code-generated values (which take defaults from the manifest).